ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -99,6 +113,7 @@
       infoText: {
         en: 'Cleanse',
         fr: 'Guérison',
+        ko: '에스나',
```

Comment:
I believe "Cleanse" means to use "Esuna" skill.
Is this right?

Reply:
I made the same assumption so I hope it correct.